posthog/hogql_queries/web_analytics/stats_table_pre_aggregated.py

```diff
@@ -174,11 +168,10 @@ def get_query(self) -> ast.SelectQuery:
                     },
                 ),
             )
-            table_name = "web_stats_combined"
 
-        filters = self._get_filters(table_name=table_name)
-        if filters:
-            query.where = filters
+            filters = self._get_filters(table_name="web_stats_combined")
+            if filters:
+                query.where = filters
```

Comment:
BTW, self._get_filters has an unusual if at the end. There's always at least 2 entries in the list, you can simplify that implementation by always returning ast.And.

Also, it always returns something, so there's no need to check `if filters` in here, you can assign it directly.

Reply:
nit: To keep this consistent, I suggest 2 things:
- extract this `else` to a separate function
- add `filters` inside the query directly even though we're not doing the `and` bit

Reply:
Makes sense, I will do those, thanks!